MailReporter/Mvc/Controllers/MailReporterController.cs

```diff
@@ -53,31 +56,43 @@ public async Task<IActionResult> Mandrill()
                 {
                     return new BadRequestObjectResult("No webhook events found");
                 }
-                //log.Info($"Processing {webhookEvents.Count} event(s)...");
-
+         
                 foreach (var webhookEvent in webhookEvents)
                 {
                     var msg = webhookEvent.Msg;
                     var mail = new NotificationEmail
                     {
                         Sender = msg.FromEmail,
-                        //Recipient = msg.
                         Subject = msg.Subject,
 
                         BodyHtml = msg.Html,
                         BodyText = msg.Text,
                     };
 
-                    var jobExecution = await jobExecutionService.ConvertMailToJobExecution(mail);
-                    //log.Info(JsonConvert.SerializeObject(jobExecution, Formatting.None));
-
-                    await jobExecutionService.Create(jobExecution);
+                    var jobExecution = await _jobExecutionService.ConvertMailToJobExecution(mail);
+                    
+                    await _jobExecutionService.Create(jobExecution);
 
                 }
             }
             catch (Exception e)
             {
-                //log.Error(e.Message + "\r\n" + e.StackTrace);
+                throw;
+            }
+            return new OkObjectResult("Event processed successfully");
+        }
+
+        [AllowAnonymous]
+        [HttpPost]
+        public async Task<IActionResult> SendInBlue(JObject payload)
+        {
+            var req = HttpContext.Request;
+            try
+            {
+                await _sendInBlueService.ProcessEvent(payload);
+            }
+            catch (Exception e)
+            {
                 throw;
             }
```

Comment:
I don't understand the reason for the catch and rethrow. Maybe it was for debugging? I think you can safely remove it.

Reply:
I will remove this try-catch and will add the logging in a separate task.